content/en/docs/concepts/storage/volumes.md

```diff
@@ -305,6 +305,11 @@ A Container using a ConfigMap as a [subPath](#using-subpath) volume mount will n
 receive ConfigMap updates.
 {{< /note >}}
 
+{{< note >}}
+Text data is exposed as files using the UTF-8 character encoding. To use some other character encoding, use binaryData.
+{{< /note >}}
```

Comment:
since this document is focused on the mount/mountPath aspect, the /docs/tasks/configure-pod-container/configure-pod-configmap/ seems like a better place for this information